File: go/fib.go
package main

import (
	"fmt"
	"time"
)

func nthFibonacci(n int) int {
	if n <= 1 {
		return n
	}
	return nthFibonacci(n-1) + nthFibonacci(n-2)
}

func main() {
	n := 40
	start := time.Now()
	_ = nthFibonacci(n)
	elapsed := time.Since(start)
	fmt.Printf("Time taken: %v seconds\n", elapsed.Seconds())
}

File: php/fib.php
<?php  
function fibonacci($n){
    if ($n <= 1) {
        return $n;
    }  
    else
        return (fibonacci($n - 1) 
              + fibonacci($n - 2));
}

// Driver Code
$n = 40;
$start = microtime(true);
$result = fibonacci($n);
$time_elapsed_secs = microtime(true) - $start;
echo "Time taken was $time_elapsed_secs seconds\n";
?>